drivers/gpio/gpio_stm32.c

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2016 Open-RnD Sp. z o.o.
+ * Copyright (c) 2018 Martin Bertsche
```

Comment:
In general we had the copyright for "significant" changes on the file, like for instance, driver restructuring or move to a new API. Even if this change is welcome, I don't think it is large enough to add a copyright on this file.

Reply:
At least worth a try :smile: